### PR TITLE
Stable helm chart version upgrade

### PR DESCRIPTION
**Pull request by kumarmadurai**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -76,6 +76,7 @@ module.exports = (security, appInsights) => {
   app.use('/logout', security.logout());
   app.use('/oauth2/callback', security.OAuth2CallbackEndpoint());
   app.use('/health/liveness', (req, res) => res.status(HttpStatus.OK).json({ status: 'UP' }));
+  app.use('/health/readiness', (req, res) => res.status(HttpStatus.OK).json({ status: 'UP' }));
```
Comment (adusumillipraveen) [inline server.js:79]: You should ideally use https://github.com/hmcts/nodejs-healthcheck module to make these changes easy. Also read https://github.com/hmcts/nodejs-healthcheck#what-to-include-in-readiness-checks for more information

File: charts/ccpay-bubble-frontend/Chart.yaml [hunk 2]
```diff
@@ -1,9 +1,17 @@
...
 name: ccpay-bubble-frontend
 home: https://github.com/hmcts/ccpay-bubble
-version: 0.2.3
+version: 0.2.4
 maintainers:
   - name: HMCTS Fees & Payments Dev Team
     email: ccpay@hmcts.net
+dependencies:
+  - name: nodejs
+    version: 2.3.2
```
Comment (adusumillipraveen) [inline charts/ccpay-bubble-frontend/Chart.yaml:12]:
```suggestion
    version: 2.3.4
```

File: charts/ccpay-bubble-frontend/Chart.yaml [hunk 3]
```diff
@@ -1,9 +1,17 @@
...
+    repository: '@hmctspublic'
+  - name: idam-pr
+    version: ~2.2.5
```
Comment (adusumillipraveen) [inline charts/ccpay-bubble-frontend/Chart.yaml:15]:
```suggestion
    version: ~2.2.6
```

File: server.js (hunk 1 above)
Comment (kumarmadurai) [reply server.js:79]: I will address this in next PR

File: charts/ccpay-bubble-frontend/Chart.yaml (hunk 3 above)
Comment (kumarmadurai) [reply charts/ccpay-bubble-frontend/Chart.yaml]: Fixed

File: charts/ccpay-bubble-frontend/Chart.yaml (hunk 2 above)
Comment (kumarmadurai) [reply charts/ccpay-bubble-frontend/Chart.yaml]: Fixed

**Review by amitchidrewar1301 — APPROVED** [review]

**Pull request closed without merging**